tools/ci_build/build.py

```diff
@@ -754,6 +758,8 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
         "-Donnxruntime_ENABLE_NVTX_PROFILE=" + ("ON" if args.enable_nvtx_profile else "OFF"),
         "-Donnxruntime_ENABLE_TRAINING=" + ("ON" if args.enable_training else "OFF"),
         "-Donnxruntime_ENABLE_TRAINING_OPS=" + ("ON" if args.enable_training_ops else "OFF"),
+        "-Donnxruntime_ENABLE_TRAINING_TORCH_INTEROP=" + ("ON" if args.enable_training_torch_interop else "OFF"),
+        "-Donnxruntime_DISABLE_UNIT_TESTS=" + ("ON" if args.disable_unit_tests else "OFF"),
```

Comment:
Some other pipelines also want to turn onnxruntime_DISABLE_UNIT_TESTS OFF. If we really need to add such an option, I suggest we make it more clear. I feel people may get confused why sometimes this option works sometimes it doesn't.

Reply:
This new flag is removed.